docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md

```diff
@@ -14,7 +14,9 @@ DROP SPACE [IF EXISTS] <graph_space_name>;
 
 You can use the `IF EXISTS` keywords when dropping spaces. These keywords automatically detect if the related graph space exists. If it exists, it will be deleted. Otherwise, no graph space will be deleted.
 
-The `DROP SPACE` statement does not remove all the files and directories from the disk. To remove all the files and directories, add `auto_remove_invalid_space=true` to the Storage Service configuration file. For the file path, see [Storage Service configurations](../../5.configurations-and-logs/1.configurations/4.storage-config.md).
+!!! compatibility "Legacy version compatibility"
+
+    In Nebula Graph versions earlier than 3.1.0, the `DROP SPACE` statement does not remove all the files and directories from the disk.
```

Comment:
the `DROP SPACE` statement does not remove all the files and directories from the disk by default.